cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelperTest.java

```diff
@@ -61,7 +61,7 @@ private void createTopics() {
   }
 
   private ExecutionTask inProgressTaskForProposal(long id, ExecutionProposal proposal) {
-    ExecutionTask task = new ExecutionTask(id, proposal, ExecutionTask.TaskType.INTER_BROKER_REPLICA_ACTION);
+    ExecutionTask task = new ExecutionTask(id, proposal, ExecutionTask.TaskType.INTER_BROKER_REPLICA_ACTION, 0);
```

Comment:
`maxExecutionTime` of 0 seems confusing. Should we ensure that there is a sanity check in `ExecutionTask` that ensures a positive value for `maxExecutionTime`?